src/main/java/frc/robot/subsystems/DriveSystem.java

```diff
@@ -81,10 +82,17 @@ public DriveSystem() {
     frontRight = new CANSparkMax(FRONT_RIGHT_MOTOR, MotorType.kBrushless);
     backRight = new CANSparkMax(BACK_RIGHT_MOTOR, MotorType.kBrushless);
 
-    frontLeft.setInverted(false);
-    frontRight.setInverted(true);
-    backLeft.setInverted(false);
-    backRight.setInverted(true);
+    if (Robot.checkType() == Robot.RobotType.A_BOT) {
+      frontLeft.setInverted(false);
+      backLeft.setInverted(false);
+      frontRight.setInverted(true);
+      backRight.setInverted(true);
+    } else {
+      frontLeft.setInverted(false);
+      frontRight.setInverted(true);
+      backLeft.setInverted(false);
+      backRight.setInverted(true);
```

Comment:
Dumb suggestion, but it would be nice if it was one-to-one with the motor controller order in the if and the else.